### ISSUE TITLE
What is the Host Name with "other"? and Why does it show the error?

### ISSUE THREAD

**Issue by Yoshihiro-jp** [ntop/ntopng#6183]
**Environment**:
* OS name: [e.g. Ubuntu] Ubuntu 
* OS version: [e.g. 18.04] 20.04.3 LTS
* Architecture: [e.g. amd64] x86_64
* ntopng version/revision: [e.g. 5.1.211223] 5.0.211220  r4205

**What happened**:
Kindly refer to the attached file.
[ntopng others problem.zip](https://github.com/ntop/ntopng/files/7836163/ntopng.others.problem.zip)

At first I choose an application in the Interface graph and then I choose the “Top Senders”.
In the “Host Name” I could find the “other”.
If I click it  the ntopng shows the error like “others2.PNG”.
If I click the Action button the ntopng shows the error like “other3.png”.
I think that this is a kind of bug.
Could you fix this problem?

**What did you expect to happen?**
That error shouldn't show when users click the link.

**How did you reproduce it?**
As mentioned above.

**Debug Information**:
Kindly refer to the attached file.

**Comment by MatteoBiscosi** [comment]
Hi @Yoshihiro-jp please try to update to the latest version and let me know if the problem persists. Now the "Other" link should be removed and a user cannot click other.

**Comment by Yoshihiro-jp** [comment]
Dear @MatteoBiscosi,

I updated below.
ntop@ntopng:~$ sudo ntopng -V
Version:        5.0.220110 [Enterprise/Professional build]
GIT rev:        5.0-stable:f9f1dd980e175bc9ea09a2dcb6bdac31781bb766:20220110
Pro rev:        r4206
Built on:       Ubuntu 20.04.3 LTS

Then I confirmed it.
For followers I wrote the your answer regarding the "other".

[What is the "Other"?]
""Other" represents an aggregation of all the other Hosts (in this case, or values in general) with a very low value, we aggregate them all together to not have (e.g.) 40 hosts with 40 bytes in the table. If you'd like to know which are the "Other" hosts, currently ntopng does not support it, we are gonna implement it in the near future. Regarding the error I'm gonna fix it, thanks for your report."


Thanks.

regards,
Yoshihiro
